hs_access_control/models/group.py

```diff
@@ -0,0 +1,235 @@
+from django.contrib.auth.models import User, Group
+from django.db import models
+from django.db.models import Q
+
+from hs_core.models import BaseResource
+from hs_access_control.models.privilege import PrivilegeCodes, UserGroupPrivilege
```

Comment:
`from hs_access_control.models.privilege import PrivilegeCodes, UserGroupPrivilege ` -> `from privilege import PrivilegeCodes, UserGroupPrivilege ` ?

Reply:
That is OK but I actually wanted to use the whole path as documentation.